pulp_ansible/app/tasks/collections.py

```diff
@@ -463,8 +511,59 @@ def _collection_list_downloader(self, api_version, collection_endpoint, page_num
             collection_list_url = f"{collection_endpoint}?limit={page_size}&offset={offset}"
         return self.remote.get_downloader(url=collection_list_url)
 
+    async def _download_unpaginated_repo_metadata(self):
+        self._unpaginated_collection_metadata = None
+        self._unpaginated_collection_version_metadata = None
```

Comment:
These should probably move to the init since code in other functions later checks them.